integrations/signup_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package integration
```

Comment:
Maybe we should move this file to `routers/user/` folder and rename to `auth_test.go` ?